data/free worlds start.txt

```diff
@@ -1418,6 +1510,68 @@ mission "FW Pirates 3.2B"
 				accept
 
 
+	
+mission "FW Pirates 3.2D"
+	name "Discretion is the Better Part"
+	description "Escape the pirate armada and report to Tomek."
+	priority
+	landing
+	to offer
+        has "FW Pirates 3.2C: active"
+	source Greenrock
+	destination Longjump
+	on fail
+        dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	on offer
+	set "fw pirates: victory"
+	dialogue
```

Comment:
You want this to be `on enter` that way it will trigger the moment you take off from the planet. Also you need to use MZ's spelling of dialog (not dialogue).